### ISSUE TITLE
As a constituent of Senator Gillibrand

### ISSUE THREAD

**Issue by RobRoseKnows** [responsible-financial-innovation-act22/RFIA-bill#89]
As a tech worker and constituent of Senator Gillibrand, I'm extremely disappointed to see such an atrocious bill with her name on it.  This Bill guts the consumer and financial protections advocates and activists have been fighting to be enforced for years now. It is a tax and liability give away to anyone who wraps a Ponzi scheme in an app and the definition of regulatory capture. While SCOTUS is on the brink of overturning Roe v. Wade, Senator Gillibrand seems to find it more important to schmooze and booze with Ponzi industry lobbyists to write a bill allowing them to scam investors with little recourse. 

I will be petitioning my union, local, district and national to withhold an endorsement of you for reelection. I also look forward to voting against you in the 2024 primary, and I hope a dedicated organizer like Cynthia Nixon or AOC runs against you. Senator Gillibrand should immediately withdraw sponsorship of this bill and return any money she has received from Ponzi industry lobbyists and their clients.

**Comment by jonmarrs** [comment]
As someone who is also in tech, I hope you can appreciate that our existing financial system is old, outdated, slow, and expensive. Consider that consumers are still using credit cards, which were invented several decades ago. People are still writing paper checks. It's 2022, and people are still using paper money and coins. The world is transitioning to digital money, and the United States needs to be a leader in this space as a major financial hub in the world. China is on a fast track to implement its digital yuan. Unless we want to live in a dystopian world where the Chinese Communist Party controls the new global reserve currency and is able to steal/freeze the funds of anyone who dares to protest or question the CCP, it is imperative that the United States puts forward a digital currency and modern financial system that reflects Western values and freedoms, based on free market principles and capitalism, that provides economic prosperity for all. Unless want China to control global trade and the global reserve currency, it is important that we allow American financial innovation to progress uninhibited.

**Comment by shikari7** [comment]
> As someone who is also in tech, I hope you can appreciate that our existing financial system is old, outdated, slow, and expensive. Consider that consumers are still using credit cards, which were invented several decades ago. People are still writing paper checks. It's 2022, and people are still using paper money and coins. The world is transitioning to digital money, and the United States needs to be a leader in this space as a major financial hub in the world. China is on a fast track to implement its digital yuan. Unless we want to live in a dystopian world where the Chinese Communist Party controls the new global reserve currency and is able to steal/freeze the funds of anyone who dares to protest or question the CCP, it is imperative that the United States puts forward a digital currency and modern financial system that reflects Western values and freedoms, based on free market principles and capitalism, that provides economic prosperity for all. Unless we want China to control global trade and the global reserve currency, it is important that we allow American financial innovation to progress uninhibited.

Regardless of truth of this, this is all completely irrelevant.  Cryptocurrencies are all Ponzi schemes.  The dollar is already digital.  You confuse the TOOLS used to trade digital monopoly "money" with the "money".  This is a common issue with crypto bros.

The United States does not need to be a leader in Ponzi schemes.  Leave that to others, like Russia.  The digital yuan is just a government controlled currency just like the yuan.  No difference.  Marketing.  "ooooo it's running on a blockchain" - there is NOTHING stopping banks or governments from running the entire financial system on blockchain, except for the fact that blockchain can't scale to that degree in an energy efficient manner.  Running on a blockchain is the tech.  Do not confuse that with the asset.  Cryptocurrency are just imaginary monies.

Also, cryptocurrency is a national security risk to all nations.  Replacing democratically elected elites with self appointed, opaque and unaccountable elites.

**Comment by CrazedCoding** [comment]
The USD is a Ponzi scheme compared to Blockchain technologies.. what typical self-elevating political spam this thread has become... You used to be able to redeem your money for gold.

 ![download (1)](https://user-images.githubusercontent.com/18431780/175662737-4e61ae52-1c75-485f-b28e-13608661bbc5.jpeg)

Proof of work is mathematical gold.

To be clear, I'm not promoting any particular cryptocurrency, but USA has been lying to itself for far too long about its own trillion dollar deficit, printing more money, spending it on all on warfare, and now that it's crashing you all think that if you take your shmear campaigns of dysfunctional logic to the internet it'll save the dollar somehow.

You kinda just gotta go to ww3 and finally get a return on investing in weapons instead of economic and political technologies to actually save the dollar... Sorry to be the one to break the news; I've been outspokenly horrified by your behavior my entire life on this issue of a dissolving democratic financial superpower into a republic dictatorship of warfare.

Now not even GitHub is safe :(

**Comment by matthewhembree** [comment]
> Politicians: please get off GitHub. It's an open source software development platform, not a soap box. Also: you code/law is always very buggy and wrecks the entire world.

I disagree with you. I want to see all bills in git. I want to see the blame for every line in a bill, so that we can expose those that add pork to otherwise sensible legislation.

**Comment by CrazedCoding** [comment]
> > Politicians: please get off GitHub. It's an open source software development platform, not a soap box. Also: you code/law is always very buggy and wrecks the entire world.
> 
> I disagree with you. I want to see all bills in git. I want to see the blame for every line in a bill, so that we can expose those that add pork to otherwise sensible legislation.

Fair position. I'm still betting they'll never start working on the machine learning related projects required to automate transparent and accountable distributed government software, and wreck anyone actually working on those (or related) technologies if they disagree.

Politicians are (violent) wastes of time, energy, and space in a domain where every bit of information counts.

**Comment by Pockets-byte** [comment]
> > > As someone who is also in tech, I hope you can appreciate that our existing financial system is old, outdated, slow, and expensive. Consider that consumers are still using credit cards, which were invented several decades ago. People are still writing paper checks. It's 2022, and people are still using paper money and coins. The world is transitioning to digital money, and the United States needs to be a leader in this space as a major financial hub in the world. China is on a fast track to implement its digital yuan. Unless we want to live in a dystopian world where the Chinese Communist Party controls the new global reserve currency and is able to steal/freeze the funds of anyone who dares to protest or question the CCP, it is imperative that the United States puts forward a digital currency and modern financial system that reflects Western values and freedoms, based on free market principles and capitalism, that provides economic prosperity for all. Unless we want China to control global trade and the global reserve currency, it is important that we allow American financial innovation to progress uninhibited.
> > 
> > 
> > Regardless of truth of this, this is all completely irrelevant.  Cryptocurrencies are all Ponzi schemes.  The dollar is already digital.  You confuse the TOOLS used to trade digital monopoly "money" with the "money".  This is a common issue with crypto bros.
> > The United States does not need to be a leader in Ponzi schemes.  Leave that to others, like Russia.  The digital yuan is just a government controlled currency just like the yuan.  No difference.  Marketing.  "ooooo it's running on a blockchain" - there is NOTHING stopping banks or governments from running the entire financial system on blockchain, except for the fact that blockchain can't scale to that degree in an energy efficient manner.  China is creating the digital yuan because they know everybody knows their yuan is rigged, so they are trying to play a bait and switch "now running on blockchain!" marketing ploy.
> > Running on a blockchain is the tech.  Do not confuse that with the asset.  Cryptocurrency are just imaginary monies.
> > Also, cryptocurrency is a national security risk to all nations.  Replacing democratically elected elites with self appointed, opaque and unaccountable elites.  This is the reason China banned cryptocurrencies.
> 
> I agree that most cryptocurrencies are scams or Ponzi schemes. However, Bitcoin is not like most other cryptocurrencies. Bitcoin is decentralized. There is no CEO or leader of Bitcoin. There are no elites in charge of Bitcoin. The Bitcoin network is completely peer-to-peer and decentralized. Bitcoin is run on >10,000 nodes which are distributed throughout the world and anyone can run a node easily and cheaply from their homes. Just download the Bitcoin software to your computer, and you can participate in the Bitcoin network and help to enforce the rules of Bitcoin (e.g., 21 million coin supply limit). Bitcoin is the first modern currency that doesn't require banks/governments/institutions/elites. With Bitcoin, you are your own bank. With Bitcoin, you can run your own node and have your own copy of the public ledger/blockchain and verify your own transactions. This decentralized nature of Bitcoin enables freedom and financial sovereignty for individuals. Bitcoin is very aligned with the American ideals of freedom and democracy. In 1776, Americans declared political independence and overthrew a tyrannical government. With Bitcoin, Americans are declaring financial independence and are overthrowing a tyrannical financial system in which bankers and elites have complete control over the money printer and bail themselves out, while Main Street is left to pay the bill with high inflation. Bitcoin fixes this with its limited currency supply and decentralized governance. So let's not confuse Bitcoin with other cryptocurrencies that are scams or Ponzi schemes. Decentralization is the key difference. Not all cryptocurrencies are created equal.

even if bitcoin is a reasonable cryptocurrency, it's ecological impact is unacceptable, and the system is excessively slow. in the end even "good" crypto is not worth it. and your real money won't magically become unusable when your internet goes out

**Comment by OyeOye-12** [comment]
> > As someone who is also in tech, I hope you can appreciate that our existing financial system is old, outdated, slow, and expensive. Consider that consumers are still using credit cards, which were invented several decades ago. People are still writing paper checks. It's 2022, and people are still using paper money and coins. The world is transitioning to digital money, and the United States needs to be a leader in this space as a major financial hub in the world. China is on a fast track to implement its digital yuan. Unless we want to live in a dystopian world where the Chinese Communist Party controls the new global reserve currency and is able to steal/freeze the funds of anyone who dares to protest or question the CCP, it is imperative that the United States puts forward a digital currency and modern financial system that reflects Western values and freedoms, based on free market principles and capitalism, that provides economic prosperity for all. Unless we want China to control global trade and the global reserve currency, it is important that we allow American financial innovation to progress uninhibited.
> 
> Regardless of truth of this, this is all completely irrelevant. Cryptocurrencies are all Ponzi schemes. The dollar is already digital. You confuse the TOOLS used to trade digital monopoly "money" with the "money". This is a common issue with crypto bros.
> 
> The United States does not need to be a leader in Ponzi schemes. Leave that to others, like Russia. The digital yuan is just a government controlled currency just like the yuan. No difference. Marketing. "ooooo it's running on a blockchain" - there is NOTHING stopping banks or governments from running the entire financial system on blockchain, except for the fact that blockchain can't scale to that degree in an energy efficient manner. China is creating the digital yuan because they know everybody knows their yuan is rigged, so they are trying to play a bait and switch "now running on blockchain!" marketing ploy. Running on a blockchain is the tech. Do not confuse that with the asset. Cryptocurrency are just imaginary monies.
> 
> Also, cryptocurrency is a national security risk to all nations. Replacing democratically elected elites with self appointed, opaque and unaccountable elites. This is the reason China banned cryptocurrencies.

All cryptos are "Ponzi Schemes" in the same way the stock market is a "Ponzi Scheme"...

**Comment by OyeOye-12** [comment]
> > > > As someone who is also in tech, I hope you can appreciate that our existing financial system is old, outdated, slow, and expensive. Consider that consumers are still using credit cards, which were invented several decades ago. People are still writing paper checks. It's 2022, and people are still using paper money and coins. The world is transitioning to digital money, and the United States needs to be a leader in this space as a major financial hub in the world. China is on a fast track to implement its digital yuan. Unless we want to live in a dystopian world where the Chinese Communist Party controls the new global reserve currency and is able to steal/freeze the funds of anyone who dares to protest or question the CCP, it is imperative that the United States puts forward a digital currency and modern financial system that reflects Western values and freedoms, based on free market principles and capitalism, that provides economic prosperity for all. Unless we want China to control global trade and the global reserve currency, it is important that we allow American financial innovation to progress uninhibited.
> > > 
> > > 
> > > Regardless of truth of this, this is all completely irrelevant.  Cryptocurrencies are all Ponzi schemes.  The dollar is already digital.  You confuse the TOOLS used to trade digital monopoly "money" with the "money".  This is a common issue with crypto bros.
> > > The United States does not need to be a leader in Ponzi schemes.  Leave that to others, like Russia.  The digital yuan is just a government controlled currency just like the yuan.  No difference.  Marketing.  "ooooo it's running on a blockchain" - there is NOTHING stopping banks or governments from running the entire financial system on blockchain, except for the fact that blockchain can't scale to that degree in an energy efficient manner.  China is creating the digital yuan because they know everybody knows their yuan is rigged, so they are trying to play a bait and switch "now running on blockchain!" marketing ploy.
> > > Running on a blockchain is the tech.  Do not confuse that with the asset.  Cryptocurrency are just imaginary monies.
> > > Also, cryptocurrency is a national security risk to all nations.  Replacing democratically elected elites with self appointed, opaque and unaccountable elites.  This is the reason China banned cryptocurrencies.
> > 
> > 
> > I agree that most cryptocurrencies are scams or Ponzi schemes. However, Bitcoin is not like most other cryptocurrencies. Bitcoin is decentralized. There is no CEO or leader of Bitcoin. There are no elites in charge of Bitcoin. The Bitcoin network is completely peer-to-peer and decentralized. Bitcoin is run on >10,000 nodes which are distributed throughout the world and anyone can run a node easily and cheaply from their homes. Just download the Bitcoin software to your computer, and you can participate in the Bitcoin network and help to enforce the rules of Bitcoin (e.g., 21 million coin supply limit). Bitcoin is the first modern currency that doesn't require banks/governments/institutions/elites. With Bitcoin, you are your own bank. With Bitcoin, you can run your own node and have your own copy of the public ledger/blockchain and verify your own transactions. This decentralized nature of Bitcoin enables freedom and financial sovereignty for individuals. Bitcoin is very aligned with the American ideals of freedom and democracy. In 1776, Americans declared political independence and overthrew a tyrannical government. With Bitcoin, Americans are declaring financial independence and are overthrowing a tyrannical financial system in which bankers and elites have complete control over the money printer and bail themselves out, while Main Street is left to pay the bill with high inflation. Bitcoin fixes this with its limited currency supply and decentralized governance. So let's not confuse Bitcoin with other cryptocurrencies that are scams or Ponzi schemes. Decentralization is the key difference. Not all cryptocurrencies are created equal.
> 
> even if bitcoin is a reasonable cryptocurrency, it's ecological impact is unacceptable, and the system is excessively slow. in the end even "good" crypto is not worth it. and your real money won't magically become unusable when your internet goes out

The ecological impact of Bitcoin is negligible compared to the traditional banking system, or even the mining industry.  You should take a look at the real numbers instead of getting caught up in hyperbole.

**Comment by jonmarrs** [comment]
![image](https://user-images.githubusercontent.com/40682667/175757670-af07a291-d265-43d5-bb57-efa33e075f07.png)

**Comment by jonmarrs** [comment]
![image](https://user-images.githubusercontent.com/40682667/175757710-b7b21ba6-658f-43a4-9fd1-5642247995c8.png)

**Comment by jonmarrs** [comment]
![image](https://user-images.githubusercontent.com/40682667/175757802-12514fa8-1fc7-4bd2-aacc-aa160195e7a8.png)

**Comment by jonmarrs** [comment]
![image](https://user-images.githubusercontent.com/40682667/175757848-55214396-712f-4afa-a98d-28404b4eab05.png)

**Comment by jonmarrs** [comment]
![image](https://user-images.githubusercontent.com/40682667/175757882-a958cf7d-b6d2-404e-8b34-c30fb49e5eb7.png)

**Comment by jonmarrs** [comment]
![image](https://user-images.githubusercontent.com/40682667/175757913-5196fdaf-95d8-40d9-8b84-669968c4f197.png)

**Comment by jonmarrs** [comment]
[Bitcoin is Key to an Abundant, Clean Energy Future](https://assets.ctfassets.net/2d5q1td6cyxq/5mRjc9X5LTXFFihIlTt7QK/e7bcba47217b60423a01a357e036105e/BCEI_White_Paper.pdf)

**Comment by jonmarrs** [comment]
[This Machine Greens](https://www.youtube.com/watch?v=b-7dMVcVWgc)

**Comment by shikari7** [comment]
> The USD is a Ponzi scheme compared to Blockchain technologies.. what typical self-elevating political spam this thread has become... You used to be able to redeem your money for gold.
> 
> ![download (1)](https://user-images.githubusercontent.com/18431780/175662737-4e61ae52-1c75-485f-b28e-13608661bbc5.jpeg)
> 
> Proof of work is mathematical gold.
> 
> To be clear, I'm not promoting any particular cryptocurrency, but USA has been lying to itself for far too long about its own trillion dollar deficit, printing more money, spending it on all on warfare, and now that it's crashing you all think that if you take your shmear campaigns of dysfunctional logic to the internet it'll save the dollar somehow.
> 
> You kinda just gotta go to ww3 and finally get a return on investing in weapons instead of economic and political technologies to actually save the dollar... Sorry to be the one to break the news; I've been outspokenly horrified by your behavior my entire life on this issue of a dissolving democratic financial superpower into a republic dictatorship of warfare.
> 
> Now not even GitHub is safe :(

hey @crazedcoding you make the typical mistake of confusing the grain exchange with the wheat.  Comparing the USD to blockchain?  Seriously...  huge gaps.  "WW3" is a scam designed to trick people into buying cryptocurrency.  Just like NFT.  There is a reason decentralization is a myth and running the WWW on blockchain is stupid - blockchain can only process 3-5 transactions per second WORLD WIDE.  It's just another scam.

Proof of work is just solving trillions of sudokus in a mass death race to generate random numbers.  Cryptocurrency is just the serial numbers from gold bars, minus the gold.  

Ponzi schemes are things that have a return on investment.  The USD is not designed to be an investment vehicle, it's designed to be a currency.  Learn the difference.  Ponzi schemes only have one source of return though - that is new investors.  That's exactly how cryptocurrency works.  The only source of return is new investors.  Hence there it is all marketing from whales and snake oil salesmen.

Inflation of the dollar is high right now but it comes and goes, and it's been basically low for over 40 years.  Doesn't change the fact that USD represents something tangible - the $300T valuation of the united states - which is generating goods and services and paying wages.  Similarly, this is why stock has value - and sometimes even pays dividends.

"crypto" (it's not) "currency" (it's not) represents NOTHING.  Digital Monopoly Money.

You cannot refute any of this.

**Comment by shikari7** [comment]
> ![image](https://user-images.githubusercontent.com/40682667/175757670-af07a291-d265-43d5-bb57-efa33e075f07.png)

And @jonmarrs those are all basically static, providing goods and services for the real economy, except Bitcoin which is a tiny fraction of the global transaction rates and is digital monopoly money and yet has energy requirements already that high and escalating exponentially.  In 100 years it will take the energy of the sun to process one transaction on the blockchain.

Back to the drawing boards with you....

**Comment by OyeOye-12** [comment]
> > ![image](https://user-images.githubusercontent.com/40682667/175757670-af07a291-d265-43d5-bb57-efa33e075f07.png)
> 
> And @jonmarrs those are all basically static, providing goods and services for the real economy, except Bitcoin which is a tiny fraction of the global transaction rates and is digital monopoly money and yet has energy requirements already that high and escalating exponentially. In 100 years it will take the energy of the sun to process one transaction on the blockchain.
> 
> Back to the drawing boards with you....

https://cointelegraph.com/news/banking-uses-56-times-more-energy-than-bitcoin-valuechain-report

Banking uses 56 times more energy than Bitcoin: Valuechain report

**Comment by shikari7** [comment]
> I agree that most cryptocurrencies are scams or Ponzi schemes. However, Bitcoin is not like most other cryptocurrencies. Bitcoin is decentralized. There is no CEO or leader of Bitcoin. There are no elites in charge of Bitcoin. The Bitcoin network is completely peer-to-peer and decentralized. Bitcoin is run on >10,000 nodes which are distributed throughout the world and anyone can run a node easily and cheaply from their homes. Just download the Bitcoin software to your computer, and you can participate in the Bitcoin network and help to enforce the rules of Bitcoin (e.g., 21 million coin supply limit). Bitcoin is the first modern currency that doesn't require banks/governments/institutions/elites. With Bitcoin, you are your own bank. With Bitcoin, you can run your own node and have your own copy of the public ledger/blockchain and verify your own transactions. This decentralized nature of Bitcoin enables freedom and financial sovereignty for individuals. Bitcoin is very aligned with the American ideals of freedom and democracy. In 1776, Americans declared political independence and overthrew a tyrannical government. With Bitcoin, Americans are declaring financial independence and are overthrowing a tyrannical financial system in which bankers and elites have complete control over the money printer and bail themselves out, while Main Street is left to pay the bill with high inflation. Bitcoin fixes this with its limited currency supply and decentralized governance. So let's not confuse Bitcoin with other cryptocurrencies that are scams or Ponzi schemes. Decentralization is the key difference. Not all cryptocurrencies are created equal.

hey @jonmarrs you are wrong all across the board.

- There is no material difference between any cryptocurrencies.  You cannot explain your statements that Bitcoin is somehow different.  That is patent nonsense.
- Cryptocurrency replaces democratically elected elites with self-appointed, opaque and unaccountable elites.  See https://bitcoinera.app/arewedecentralizedyet/
- Bitcoin (and all cryptocurrencies) is digital monopoly money. No intrinsic utility. This means it is inherently a Ponzi scheme.  The only source of return on your investment is from new investors.  That's the definition.
- Cryptocurrency is a national security risk. Why would anyone want to hand over control of their economy and fiscal policy to unaccountable self-appointed elites who are only in it to enrich themselves? (see the link above again)

**Comment by shikari7** [comment]
> Banking uses 56 times more energy than Bitcoin: Valuechain report

hey @OyeOye-12 as I noted elsewhere, this is irrelevant. It uses as much energy as Switzerland just solving trillions of sudokus in a death race to generate random numbers.  That's all cryptocurrency is, random numbers.  And that energy usage is growing exponentially.  In 100 years it will require the energy output of the Sun to process one transaction on the blockchain.  A blockchain which can only process 4-5 txns/sec GLOBALLY.  And only getting slower as the blockchain grows.

**Comment by OyeOye-12** [comment]
> > Banking uses 56 times more energy than Bitcoin: Valuechain report
> 
> hey @OyeOye-12 as I noted elsewhere, this is irrelevant. It uses as much energy as Switzerland just solving trillions of sudokus in a death race to generate random numbers. That's all cryptocurrency is, random numbers. And that energy usage is growing exponentially. In 100 years it will require the energy output of the Sun to process one transaction on the blockchain. A blockchain which can only process 4-5 txns/sec GLOBALLY. And only getting slower as the blockchain grows.

I think it is quite relevant, as many detractors like to argue that Bitcoin is not "green" enough.  Clearly it is.  Bitcoin also transacts a large amount of value globally, and has done so for 13+ years now.

**Comment by CrazedCoding** [comment]
> "crypto" (it's not) "currency" (it's not) represents NOTHING. Digital Monopoly Money.
> 
> You cannot refute any of this.

How about: I started with the conclusion, and immediately realized how arrogant the entire point of the post was...

![nobody_cares](https://user-images.githubusercontent.com/18431780/176081803-e1b36fa2-8dfe-420e-80f6-799bebabdc1e.png)

TLDR;

**Comment by CrazedCoding** [comment]
> > "crypto" (it's not) "currency" (it's not) represents NOTHING. Digital Monopoly Money.
> > You cannot refute any of this.
> 
> How about: I started with the conclusion, and immediately realized how arrogant the entire point of the post was...
> 
> ![nobody_cares](https://user-images.githubusercontent.com/18431780/176081803-e1b36fa2-8dfe-420e-80f6-799bebabdc1e.png)
> 
> TLDR;

Knowledge is power, power corrupts, but only absolute knowledge corrupts absolutely.

**Comment by CrazedCoding** [comment]
> > > Politicians: please get off GitHub. It's an open source software development platform, not a soap box. Also: you code/law is always very buggy and wrecks the entire world.
> > 
> > 
> > I disagree with you. I want to see all bills in git. I want to see the blame for every line in a bill, so that we can expose those that add pork to otherwise sensible legislation.
> 
> Fair position. I'm still betting they'll never start working on the machine learning related projects required to automate transparent and accountable distributed government software, and wreck anyone actually working on those (or related) technologies if they disagree.
> 
> Politicians are (violent) wastes of time, energy, and space in a domain where every bit of information counts.

Bump.

**Comment by CrazedCoding** [comment]
> > > "crypto" (it's not) "currency" (it's not) represents NOTHING. Digital Monopoly Money.
> > > You cannot refute any of this.
> > 
> > 
> > How about: I started with the conclusion, and immediately realized how arrogant the entire point of the post was...
> > ![nobody_cares](https://user-images.githubusercontent.com/18431780/176081803-e1b36fa2-8dfe-420e-80f6-799bebabdc1e.png)
> > TLDR;
> 
> Knowledge is power, power corrupts, but only absolute knowledge corrupts absolutely.

Mentioned here: https://github.com/responsible-financial-innovation-act22/RFIA-bill/issues/138#issuecomment-1168149528

**Comment by shikari7** [comment]
> > > Banking uses 56 times more energy than Bitcoin: Valuechain report
> > 
> > 
> > hey @OyeOye-12 as I noted elsewhere, this is irrelevant. It uses as much energy as Switzerland just solving trillions of sudokus in a death race to generate random numbers. That's all cryptocurrency is, random numbers. And that energy usage is growing exponentially. In 100 years it will require the energy output of the Sun to process one transaction on the blockchain. A blockchain which can only process 4-5 txns/sec GLOBALLY. And only getting slower as the blockchain grows.
> 
> I think it is quite relevant, as many detractors like to argue that Bitcoin is not "green" enough. Clearly it is. Bitcoin also transacts a large amount of value globally, and has done so for 13+ years now.

As I pointed out @OyeOye-12 you seem to have missed - Bitcoin does this only to solve trillions of Sudokus in a death race to generate random numbers, and this is escalating exponentially as the pool of Bitcoin dissipates over time.  Bitcoin is not green at all and it's accelerating away from being green.  AND all this for a max of 4-5 transactions per SECOND globally.  

You need to think long and hard about that before you respond.  And then directly address these points instead of more hand waving.

**Comment by shikari7** [comment]
> > "crypto" (it's not) "currency" (it's not) represents NOTHING. Digital Monopoly Money.
> > You cannot refute any of this.
> 
> How about: I started with the conclusion, and immediately realized how arrogant the entire point of the post was...
> 
> ![nobody_cares](https://user-images.githubusercontent.com/18431780/176081803-e1b36fa2-8dfe-420e-80f6-799bebabdc1e.png)

Says @CrazedCoding who just proved through making typical excuses of a weak mind that he cannot refute anything I said. As fully predicted.

> > > > Politicians: please get off GitHub. It's an open source software development platform, not a soap box. Also: you code/law is always very buggy and wrecks the entire world.
> > > 
> > > I disagree with you. I want to see all bills in git. I want to see the blame for every line in a bill, so that we can expose those that add pork to otherwise sensible legislation.
> > 
> > Fair position. I'm still betting they'll never start working on the machine learning related projects required to automate transparent and accountable distributed government software, and wreck anyone actually working on those (or related) technologies if they disagree.
> > Politicians are (violent) wastes of time, energy, and space in a domain where every bit of information counts.

This @CrazedCoding is a braindead comment.  Politicians are inevitable just like government is inevitable.  Grow up.

**Comment by shikari7** [comment]
> All cryptos are "Ponzi Schemes" in the same way the stock market is a "Ponzi Scheme"...

Revealing incredible ignorance about finance, currency and math.  As I explained elsewhere, stock have intrinsic utility.  They inherently represent a creative concern that is producing goods and services, and paying wages.

All cryptos do not represent anything.  They are just random numbers stored in someone else's database.

Yet another thing @OyeOye-12 cannot refute but will just make more excuses to try to change the subject...
